tidb-cloud/backup-and-restore.md

```diff
@@ -91,8 +91,9 @@ To configure the backup schedule, perform the following steps:
 
         > **Note**
         >
-        > - After you delete a cluster, the automatic backup files will be retained for a specified period, as set in backup retention. You need to delete the backup files accordingly.
-        > - After you delete a cluster, the existing manual backup files will be retained until you manually delete them, or your account is closed.
+        > - All auto-backups, except the latest one, will be deleted if their lifetime exceeds the retention period. The latest auto-backup will not be deleted unless you delete it explicitly. This ensures that a cluster still has data to restore after it is accidentally deleted.
+        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the Recycle Bin. 
```

Comment:
```suggestion
        > - After you delete a cluster, auto-backups with a lifetime within the retention period will be moved to the recycle bin.
```
Keep the capitalization consistent with:
https://github.com/pingcap/docs/blob/a0100a3605f5e062367e1d8d3b3017c412986429/tidb-cloud/backup-and-restore.md?plain=1#L9